_posts/2017-07-21-pydelhi-conf-2017-a-beautiful-conference-happend-at-new-delhi-india.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: post
+title: "Pydelhi Conf 2017: A beautiful conference happened in New Delhi, India"
+date: "2017-07-21 18:14:03 +0530"
+tag:
+  - python
+  - conference
+  - django
+  - talks
+---
+
+![Pydelhi Conf
+2017]({{site.url}}/assets/images/pydelhi_conf_2017/group_photo.jpg)
+
+## TL;DR
+
+[PyDelhi conf 2017][pydelhi_conf_2017] was a two-day conference which served
+talks, lightning talks, workshops and dev sprint. If you are a beginner,
+attending this conference is worth because there are free workshops. If you are
+an experienced, attending this conference is worth because of curated advanced
```

Comment:
`experienced` is not a noun; try either just `experienced` or `an experienced developer`. Same comment on `worth` above applied here. `curated advanced level talks` should be `curated, advanced-level talks`.